test/specs/collections/Form/Form-test.js

```diff
@@ -44,21 +44,42 @@ describe('Form', () => {
 
   describe('onSubmit', () => {
     it('prevents default on the event when there is no action', () => {
-      const event = { preventDefault: sandbox.spy() }
+      const preventDefault = sandbox.spy()
 
-      shallow(<Form onSubmit={sandbox.spy()} />)
-        .simulate('submit', event)
-
-      event.preventDefault.should.have.been.calledOnce()
+      shallow(<Form onSubmit={_.noop} />)
+        .simulate('submit', { preventDefault })
+      preventDefault.should.have.been.calledOnce()
```

Comment:
This is OK for this PR but I'd prefer we don't make arbitrary refactors such as this.  It helps keeps the PRs smaller.

An exception would be refactors for consistency in coding standards, however, we use `const event = {}` quite a bit in tests.